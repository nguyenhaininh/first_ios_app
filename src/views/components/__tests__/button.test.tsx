/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import Button from '../button';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const onPress = jest.fn();
  const tree = renderer.create(<Button onPress={onPress}>Test</Button>).toJSON();
  expect(tree).toMatchSnapshot();
});
