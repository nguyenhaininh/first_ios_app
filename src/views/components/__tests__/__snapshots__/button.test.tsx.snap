// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#004098",
      "borderRadius": 4,
      "minHeight": 42,
      "opacity": 1,
      "paddingBottom": 13,
      "paddingTop": 13,
      "width": 200,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontSize": 16,
          "fontWeight": "bold",
        },
        undefined,
        undefined,
      ]
    }
  >
    Test
  </Text>
</View>
`;
